src/assets/TokensController.ts

```diff
@@ -326,7 +326,10 @@ export class TokensController extends BaseController<
     // to check against the contract
     if (contractsMap[checksumAddress]?.erc721 === true) {
       return Promise.resolve(true);
+    } else if (contractsMap[checksumAddress]?.erc20 === true) {
+      return Promise.resolve(false);
     }
```

Comment:
@Gudahtt  I've add this block to avoid unnecessary network requests in this method when we know the token is ERC20 because its in our contract map.

Reply:
Good idea!